Master build break: TestSecurityVulnerabilities

Failing packages:

- gcc-5-base
- libstdc++6

`Inst gcc-5-base [5.4.0-6ubuntu1\~16.04.9] (5.4.0-6ubuntu1\~16.04.10 Ubuntu:16.04/xenial-security [amd64]) [libstdc++6:amd64 ]`
`Conf gcc-5-base (5.4.0-6ubuntu1\~16.04.10 Ubuntu:16.04/xenial-security [amd64]) [libstdc++6:amd64 ]`

`Inst libstdc++6 [5.4.0-6ubuntu1\~16.04.9] (5.4.0-6ubuntu1\~16.04.10 Ubuntu:16.04/xenial-security [amd64])`
`Conf libstdc++6 (5.4.0-6ubuntu1\~16.04.10 Ubuntu:16.04/xenial-security [amd64])`

Raw test output
```
[34m[1m[4mTest mqadvanced-server-dev:9.0.5.0-x86_64-ubuntu-16.04 on Docker version 18.05.0-ce, build f150324[0m
cd test/docker && TEST_IMAGE=mqadvanced-server-dev:9.0.5.0-x86_64-ubuntu-16.04 EXPECTED_LICENSE=Developer DEV_JMS_IMAGE=mq-dev-jms-test go test -parallel 2 -tags mqdev 
--- FAIL: TestSecurityVulnerabilities (3.00s)
	docker_api_test_util.go:241: Running container (mqadvanced-server-dev:9.0.5.0-x86_64-ubuntu-16.04)
	docker_api_test_util.go:267: Starting container: f65a05dc90c192b0f3983f38f7739798e99b9d45d0b811289f55a821af26dee9
	docker_api_test_util.go:176: Inspected container f65a05dc90c192b0f3983f38f7739798e99b9d45d0b811289f55a821af26dee9: types.ContainerJSON{ContainerJSONBase:(*types.ContainerJSONBase)(0xc4201b69a0), Mounts:[]types.MountPoint{types.MountPoint{Type:"bind", Name:"", Source:"/home/travis/gopath/src/github.com/ibm-messaging/mq-container/test/docker/coverage", Destination:"/var/coverage", Driver:"", Mode:"", RW:true, Propagation:"rprivate"}, types.MountPoint{Type:"bind", Name:"", Source:"/tmp/TestSecurityVulnerabilities-termination-log", Destination:"/dev/termination-log", Driver:"", Mode:"", RW:true, Propagation:"rprivate"}}, Config:(*container.Config)(0xc4200e8000), NetworkSettings:(*types.NetworkSettings)(0xc4201a6700)}
	docker_api_test_util.go:181: Inspected container f65a05dc90c192b0f3983f38f7739798e99b9d45d0b811289f55a821af26dee9: {
		    "Id": "f65a05dc90c192b0f3983f38f7739798e99b9d45d0b811289f55a821af26dee9",
		    "Created": "2018-06-28T10:12:42.034636941Z",
		    "Path": "bash",
		    "Args": [
		        "-c",
		        "test -d /etc/apt"
		    ],
		    "State": {
		        "Status": "exited",
		        "Running": false,
		        "Paused": false,
		        "Restarting": false,
		        "OOMKilled": false,
		        "Dead": false,
		        "Pid": 0,
		        "ExitCode": 0,
		        "Error": "",
		        "StartedAt": "2018-06-28T10:12:42.400803008Z",
		        "FinishedAt": "2018-06-28T10:12:42.443692921Z"
		    },
		    "Image": "sha256:cb1fa659a264397ff237a977d7e0188cfd2ac56da130f968a9cc286760e3a013",
		    "ResolvConfPath": "/var/lib/docker/containers/f65a05dc90c192b0f3983f38f7739798e99b9d45d0b811289f55a821af26dee9/resolv.conf",
		    "HostnamePath": "/var/lib/docker/containers/f65a05dc90c192b0f3983f38f7739798e99b9d45d0b811289f55a821af26dee9/hostname",
		    "HostsPath": "/var/lib/docker/containers/f65a05dc90c192b0f3983f38f7739798e99b9d45d0b811289f55a821af26dee9/hosts",
		    "LogPath": "/var/lib/docker/containers/f65a05dc90c192b0f3983f38f7739798e99b9d45d0b811289f55a821af26dee9/f65a05dc90c192b0f3983f38f7739798e99b9d45d0b811289f55a821af26dee9-json.log",
		    "Name": "/TestSecurityVulnerabilities",
		    "RestartCount": 0,
		    "Driver": "overlay2",
		    "MountLabel": "",
		    "ProcessLabel": "",
		    "AppArmorProfile": "",
		    "ExecIDs": null,
		    "HostConfig": {
		        "Binds": [
		            "/home/travis/gopath/src/github.com/ibm-messaging/mq-container/test/docker/coverage:/var/coverage",
		            "/tmp/TestSecurityVulnerabilities-termination-log:/dev/termination-log"
		        ],
		        "ContainerIDFile": "",
		        "LogConfig": {
		            "Type": "json-file",
		            "Config": {}
		        },
		        "NetworkMode": "default",
		        "PortBindings": {},
		        "RestartPolicy": {
		            "Name": "",
		            "MaximumRetryCount": 0
		        },
		        "AutoRemove": false,
		        "VolumeDriver": "",
		        "VolumesFrom": null,
		        "CapAdd": null,
		        "CapDrop": null,
		        "Dns": null,
		        "DnsOptions": null,
		        "DnsSearch": null,
		        "ExtraHosts": null,
		        "GroupAdd": null,
		        "IpcMode": "shareable",
		        "Cgroup": "",
		        "Links": null,
		        "OomScoreAdj": 0,
		        "PidMode": "",
		        "Privileged": false,
		        "PublishAllPorts": false,
		        "ReadonlyRootfs": false,
		        "SecurityOpt": null,
		        "UTSMode": "",
		        "UsernsMode": "",
		        "ShmSize": 67108864,
		        "Runtime": "runc",
		        "ConsoleSize": [
		            0,
		            0
		        ],
		        "Isolation": "",
		        "CpuShares": 0,
		        "Memory": 0,
		        "NanoCpus": 0,
		        "CgroupParent": "",
		        "BlkioWeight": 0,
		        "BlkioWeightDevice": null,
		        "BlkioDeviceReadBps": null,
		        "BlkioDeviceWriteBps": null,
		        "BlkioDeviceReadIOps": null,
		        "BlkioDeviceWriteIOps": null,
		        "CpuPeriod": 0,
		        "CpuQuota": 0,
		        "CpuRealtimePeriod": 0,
		        "CpuRealtimeRuntime": 0,
		        "CpusetCpus": "",
		        "CpusetMems": "",
		        "Devices": null,
		        "DiskQuota": 0,
		        "KernelMemory": 0,
		        "MemoryReservation": 0,
		        "MemorySwap": 0,
		        "MemorySwappiness": null,
		        "OomKillDisable": false,
		        "PidsLimit": 0,
		        "Ulimits": null,
		        "CpuCount": 0,
		        "CpuPercent": 0,
		        "IOMaximumIOps": 0,
		        "IOMaximumBandwidth": 0
		    },
		    "GraphDriver": {
		        "Name": "overlay2",
		        "Data": {
		            "LowerDir": "/var/lib/docker/overlay2/0a9ea3119177fd97fe22d358592528eaf3e040f57877120aba954c0ee547b9d8-init/diff:/var/lib/docker/overlay2/d7eef2af9da5d2ce0ac43d7298d4be6b4c01e624ca9046ed6db032d93d517d65/diff:/var/lib/docker/overlay2/2efeef6cbab303e95e3102d84f9a78c5982ad7076ef8c9cdadff2c1cc688de15/diff:/var/lib/docker/overlay2/dc1f50ef05ee78c204d2e74f0df72283efd6d2279b2d288b2c93940b8ffb5d5b/diff:/var/lib/docker/overlay2/dd1c51912cdefd6fbf2855224fdde905110864886f897a8a98a6a60b803fa2a7/diff:/var/lib/docker/overlay2/a9180dcf1e5f153906b869b4ea826dc1505ad75a572a4dff0eb5d0615781ecc9/diff:/var/lib/docker/overlay2/4ecbd385138fc1c2b1018a60d9927103015d01bf63b677e4661619a92ad5ed99/diff:/var/lib/docker/overlay2/2ced635755b44641e42db3ab35ae99f10a84be633db57dc329a6960b00230829/diff:/var/lib/docker/overlay2/ce6f651904551c7dd1f7d43a13d3aca935edd55874021bb6ad2ea339171c1ec5/diff:/var/lib/docker/overlay2/094329c596234d433b23c0efce9499cd0615fda24e48e8c09850c5689d1f734d/diff:/var/lib/docker/overlay2/2826d6b61a8b6874f9380155cbad77a11c6b2ce83f85d8801bdb49fc066ff908/diff:/var/lib/docker/overlay2/3f4c3bd04e05c1b8c2faf06713abdb11b5b499560d3b2c44ecb3a9172176db13/diff:/var/lib/docker/overlay2/eaf75f728cf45efde479c68005829b8b43e8ddf457f2abac385ee3c2c2199efc/diff:/var/lib/docker/overlay2/5b4165dffb5e2b711ce1e3627709794d79f0c347c726fbb3939e57c29db6d0ed/diff:/var/lib/docker/overlay2/8cd831f3d475960de94fe473675f636ae20db85fe3caf02b1688925ac6f07784/diff:/var/lib/docker/overlay2/3aa78a468a1415d14f950b4a40a62356a23bef2ca80556de5e6d86175c389f95/diff:/var/lib/docker/overlay2/edba18d11daa221e873dc0cafd4d76325090394929877feecf8126b6da19b4bc/diff:/var/lib/docker/overlay2/5bccb062e9ff1cd164091d6124fe5d3285255d8476951817192cd9a6e15b6c30/diff:/var/lib/docker/overlay2/a21bdeeb10e6fd3a7d7388f641f66426a7faee979b45d723de0368f6a8b2f129/diff:/var/lib/docker/overlay2/96c2f2a31da2714ceae99ba2eca9b841c9cb6555e236f8933412cda3a35e6252/diff",
		            "MergedDir": "/var/lib/docker/overlay2/0a9ea3119177fd97fe22d358592528eaf3e040f57877120aba954c0ee547b9d8/merged",
		            "UpperDir": "/var/lib/docker/overlay2/0a9ea3119177fd97fe22d358592528eaf3e040f57877120aba954c0ee547b9d8/diff",
		            "WorkDir": "/var/lib/docker/overlay2/0a9ea3119177fd97fe22d358592528eaf3e040f57877120aba954c0ee547b9d8/work"
		        }
		    },
		    "Mounts": [
		        {
		            "Type": "bind",
		            "Source": "/home/travis/gopath/src/github.com/ibm-messaging/mq-container/test/docker/coverage",
		            "Destination": "/var/coverage",
		            "Mode": "",
		            "RW": true,
		            "Propagation": "rprivate"
		        },
		        {
		            "Type": "bind",
		            "Source": "/tmp/TestSecurityVulnerabilities-termination-log",
		            "Destination": "/dev/termination-log",
		            "Mode": "",
		            "RW": true,
		            "Propagation": "rprivate"
		        }
		    ],
		    "Config": {
		        "Hostname": "f65a05dc90c1",
		        "Domainname": "",
		        "User": "",
		        "AttachStdin": false,
		        "AttachStdout": false,
		        "AttachStderr": false,
		        "ExposedPorts": {
		            "1414/tcp": {},
		            "9157/tcp": {},
		            "9443/tcp": {}
		        },
		        "Tty": false,
		        "OpenStdin": false,
		        "StdinOnce": false,
		        "Env": [
		            "COVERAGE_FILE=TestSecurityVulnerabilities.cov",
		            "EXIT_CODE_FILE=TestSecurityVulnerabilitiesExitCode",
		            "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
		            "LANG=en_US.UTF-8",
		            "AMQ_DIAGNOSTIC_MSG_SEVERITY=1",
		            "AMQ_ADDITIONAL_JSON_LOG=1",
		            "LOG_FORMAT=basic",
		            "MQ_DEV=true",
		            "MQ_ADMIN_PASSWORD=passw0rd"
		        ],
		        "Cmd": null,
		        "Image": "mqadvanced-server-dev:9.0.5.0-x86_64-ubuntu-16.04",
		        "Volumes": null,
		        "WorkingDir": "",
		        "Entrypoint": [
		            "bash",
		            "-c",
		            "test -d /etc/apt"
		        ],
		        "OnBuild": null,
		        "Labels": {
		            "IBM_PRODUCT_ID": "98102d16795c4263ad9ca075190a2d4d",
		            "IBM_PRODUCT_NAME": "IBM MQ Advanced for Developers (Non-Warranted)",
		            "IBM_PRODUCT_VERSION": "9.0.5.0"
		        }
		    },
		    "NetworkSettings": {
		        "Bridge": "",
		        "SandboxID": "3142b5cfa11cda0b6dfa712d0efcbf9028efd0c4e0b5221e806acf45f31f3e32",
		        "HairpinMode": false,
		        "LinkLocalIPv6Address": "",
		        "LinkLocalIPv6PrefixLen": 0,
		        "Ports": {},
		        "SandboxKey": "/var/run/docker/netns/3142b5cfa11c",
		        "SecondaryIPAddresses": null,
		        "SecondaryIPv6Addresses": null,
		        "EndpointID": "",
		        "Gateway": "",
		        "GlobalIPv6Address": "",
		        "GlobalIPv6PrefixLen": 0,
		        "IPAddress": "",
		        "IPPrefixLen": 0,
		        "IPv6Gateway": "",
		        "MacAddress": "",
		        "Networks": {
		            "bridge": {
		                "IPAMConfig": null,
		                "Links": null,
		                "Aliases": null,
		                "NetworkID": "b0f7ebe61a150e0e336d8ed3c431e30eda620dc442413ea0ca78f41334878846",
		                "EndpointID": "",
		                "Gateway": "",
		                "IPAddress": "",
		                "IPPrefixLen": 0,
		                "IPv6Gateway": "",
		                "GlobalIPv6Address": "",
		                "GlobalIPv6PrefixLen": 0,
		                "MacAddress": ""
		            }
		        }
		    }
		}
	docker_api_test_util.go:183: Stopping container: f65a05dc90c192b0f3983f38f7739798e99b9d45d0b811289f55a821af26dee9
	docker_api_test_util.go:191: Container stopped
	docker_api_test_util.go:196: Console log from container f65a05dc90c192b0f3983f38f7739798e99b9d45d0b811289f55a821af26dee9:
		
	docker_api_test_util.go:204: Removing container: f65a05dc90c192b0f3983f38f7739798e99b9d45d0b811289f55a821af26dee9
	docker_api_test_util.go:241: Running container (mqadvanced-server-dev:9.0.5.0-x86_64-ubuntu-16.04)
	docker_api_test_util.go:267: Starting container: e23371d35b8eb0315e9e1efde1121f124075c4b204f6ab4a76f9f2da01fd1732
	docker_api_test_util.go:176: Inspected container e23371d35b8eb0315e9e1efde1121f124075c4b204f6ab4a76f9f2da01fd1732: types.ContainerJSON{ContainerJSONBase:(*types.ContainerJSONBase)(0xc4202ca6e0), Mounts:[]types.MountPoint{types.MountPoint{Type:"bind", Name:"", Source:"/home/travis/gopath/src/github.com/ibm-messaging/mq-container/test/docker/coverage", Destination:"/var/coverage", Driver:"", Mode:"", RW:true, Propagation:"rprivate"}, types.MountPoint{Type:"bind", Name:"", Source:"/tmp/TestSecurityVulnerabilities-termination-log", Destination:"/dev/termination-log", Driver:"", Mode:"", RW:true, Propagation:"rprivate"}}, Config:(*container.Config)(0xc4200e8000), NetworkSettings:(*types.NetworkSettings)(0xc4201b5b00)}
	docker_api_test_util.go:181: Inspected container e23371d35b8eb0315e9e1efde1121f124075c4b204f6ab4a76f9f2da01fd1732: {
		    "Id": "e23371d35b8eb0315e9e1efde1121f124075c4b204f6ab4a76f9f2da01fd1732",
		    "Created": "2018-06-28T10:12:42.667282562Z",
		    "Path": "bash",
		    "Args": [
		        "-c",
		        "source /etc/os-release \u0026\u0026 echo \"deb http://security.ubuntu.com/ubuntu/ ${VERSION_CODENAME}-security main restricted\" \u003e /etc/apt/sources.list \u0026\u0026 apt-get update 2\u003e\u00261 \u003e/dev/null \u0026\u0026 apt-get --simulate -qq upgrade"
		    ],
		    "State": {
		        "Status": "exited",
		        "Running": false,
		        "Paused": false,
		        "Restarting": false,
		        "OOMKilled": false,
		        "Dead": false,
		        "Pid": 0,
		        "ExitCode": 0,
		        "Error": "",
		        "StartedAt": "2018-06-28T10:12:43.222295779Z",
		        "FinishedAt": "2018-06-28T10:12:44.748793995Z"
		    },
		    "Image": "sha256:cb1fa659a264397ff237a977d7e0188cfd2ac56da130f968a9cc286760e3a013",
		    "ResolvConfPath": "/var/lib/docker/containers/e23371d35b8eb0315e9e1efde1121f124075c4b204f6ab4a76f9f2da01fd1732/resolv.conf",
		    "HostnamePath": "/var/lib/docker/containers/e23371d35b8eb0315e9e1efde1121f124075c4b204f6ab4a76f9f2da01fd1732/hostname",
		    "HostsPath": "/var/lib/docker/containers/e23371d35b8eb0315e9e1efde1121f124075c4b204f6ab4a76f9f2da01fd1732/hosts",
		    "LogPath": "/var/lib/docker/containers/e23371d35b8eb0315e9e1efde1121f124075c4b204f6ab4a76f9f2da01fd1732/e23371d35b8eb0315e9e1efde1121f124075c4b204f6ab4a76f9f2da01fd1732-json.log",
		    "Name": "/TestSecurityVulnerabilities",
		    "RestartCount": 0,
		    "Driver": "overlay2",
		    "MountLabel": "",
		    "ProcessLabel": "",
		    "AppArmorProfile": "",
		    "ExecIDs": null,
		    "HostConfig": {
		        "Binds": [
		            "/home/travis/gopath/src/github.com/ibm-messaging/mq-container/test/docker/coverage:/var/coverage",
		            "/tmp/TestSecurityVulnerabilities-termination-log:/dev/termination-log"
		        ],
		        "ContainerIDFile": "",
		        "LogConfig": {
		            "Type": "json-file",
		            "Config": {}
		        },
		        "NetworkMode": "default",
		        "PortBindings": {},
		        "RestartPolicy": {
		            "Name": "",
		            "MaximumRetryCount": 0
		        },
		        "AutoRemove": false,
		        "VolumeDriver": "",
		        "VolumesFrom": null,
		        "CapAdd": null,
		        "CapDrop": null,
		        "Dns": null,
		        "DnsOptions": null,
		        "DnsSearch": null,
		        "ExtraHosts": null,
		        "GroupAdd": null,
		        "IpcMode": "shareable",
		        "Cgroup": "",
		        "Links": null,
		        "OomScoreAdj": 0,
		        "PidMode": "",
		        "Privileged": false,
		        "PublishAllPorts": false,
		        "ReadonlyRootfs": false,
		        "SecurityOpt": null,
		        "UTSMode": "",
		        "UsernsMode": "",
		        "ShmSize": 67108864,
		        "Runtime": "runc",
		        "ConsoleSize": [
		            0,
		            0
		        ],
		        "Isolation": "",
		        "CpuShares": 0,
		        "Memory": 0,
		        "NanoCpus": 0,
		        "CgroupParent": "",
		        "BlkioWeight": 0,
		        "BlkioWeightDevice": null,
		        "BlkioDeviceReadBps": null,
		        "BlkioDeviceWriteBps": null,
		        "BlkioDeviceReadIOps": null,
		        "BlkioDeviceWriteIOps": null,
		        "CpuPeriod": 0,
		        "CpuQuota": 0,
		        "CpuRealtimePeriod": 0,
		        "CpuRealtimeRuntime": 0,
		        "CpusetCpus": "",
		        "CpusetMems": "",
		        "Devices": null,
		        "DiskQuota": 0,
		        "KernelMemory": 0,
		        "MemoryReservation": 0,
		        "MemorySwap": 0,
		        "MemorySwappiness": null,
		        "OomKillDisable": false,
		        "PidsLimit": 0,
		        "Ulimits": null,
		        "CpuCount": 0,
		        "CpuPercent": 0,
		        "IOMaximumIOps": 0,
		        "IOMaximumBandwidth": 0
		    },
		    "GraphDriver": {
		        "Name": "overlay2",
		        "Data": {
		            "LowerDir": "/var/lib/docker/overlay2/29018cb311ab1779bb1743bd63e7ad02cf348ebf7fdf47be9adef39b1c2cb9ab-init/diff:/var/lib/docker/overlay2/d7eef2af9da5d2ce0ac43d7298d4be6b4c01e624ca9046ed6db032d93d517d65/diff:/var/lib/docker/overlay2/2efeef6cbab303e95e3102d84f9a78c5982ad7076ef8c9cdadff2c1cc688de15/diff:/var/lib/docker/overlay2/dc1f50ef05ee78c204d2e74f0df72283efd6d2279b2d288b2c93940b8ffb5d5b/diff:/var/lib/docker/overlay2/dd1c51912cdefd6fbf2855224fdde905110864886f897a8a98a6a60b803fa2a7/diff:/var/lib/docker/overlay2/a9180dcf1e5f153906b869b4ea826dc1505ad75a572a4dff0eb5d0615781ecc9/diff:/var/lib/docker/overlay2/4ecbd385138fc1c2b1018a60d9927103015d01bf63b677e4661619a92ad5ed99/diff:/var/lib/docker/overlay2/2ced635755b44641e42db3ab35ae99f10a84be633db57dc329a6960b00230829/diff:/var/lib/docker/overlay2/ce6f651904551c7dd1f7d43a13d3aca935edd55874021bb6ad2ea339171c1ec5/diff:/var/lib/docker/overlay2/094329c596234d433b23c0efce9499cd0615fda24e48e8c09850c5689d1f734d/diff:/var/lib/docker/overlay2/2826d6b61a8b6874f9380155cbad77a11c6b2ce83f85d8801bdb49fc066ff908/diff:/var/lib/docker/overlay2/3f4c3bd04e05c1b8c2faf06713abdb11b5b499560d3b2c44ecb3a9172176db13/diff:/var/lib/docker/overlay2/eaf75f728cf45efde479c68005829b8b43e8ddf457f2abac385ee3c2c2199efc/diff:/var/lib/docker/overlay2/5b4165dffb5e2b711ce1e3627709794d79f0c347c726fbb3939e57c29db6d0ed/diff:/var/lib/docker/overlay2/8cd831f3d475960de94fe473675f636ae20db85fe3caf02b1688925ac6f07784/diff:/var/lib/docker/overlay2/3aa78a468a1415d14f950b4a40a62356a23bef2ca80556de5e6d86175c389f95/diff:/var/lib/docker/overlay2/edba18d11daa221e873dc0cafd4d76325090394929877feecf8126b6da19b4bc/diff:/var/lib/docker/overlay2/5bccb062e9ff1cd164091d6124fe5d3285255d8476951817192cd9a6e15b6c30/diff:/var/lib/docker/overlay2/a21bdeeb10e6fd3a7d7388f641f66426a7faee979b45d723de0368f6a8b2f129/diff:/var/lib/docker/overlay2/96c2f2a31da2714ceae99ba2eca9b841c9cb6555e236f8933412cda3a35e6252/diff",
		            "MergedDir": "/var/lib/docker/overlay2/29018cb311ab1779bb1743bd63e7ad02cf348ebf7fdf47be9adef39b1c2cb9ab/merged",
		            "UpperDir": "/var/lib/docker/overlay2/29018cb311ab1779bb1743bd63e7ad02cf348ebf7fdf47be9adef39b1c2cb9ab/diff",
		            "WorkDir": "/var/lib/docker/overlay2/29018cb311ab1779bb1743bd63e7ad02cf348ebf7fdf47be9adef39b1c2cb9ab/work"
		        }
		    },
		    "Mounts": [
		        {
		            "Type": "bind",
		            "Source": "/home/travis/gopath/src/github.com/ibm-messaging/mq-container/test/docker/coverage",
		            "Destination": "/var/coverage",
		            "Mode": "",
		            "RW": true,
		            "Propagation": "rprivate"
		        },
		        {
		            "Type": "bind",
		            "Source": "/tmp/TestSecurityVulnerabilities-termination-log",
		            "Destination": "/dev/termination-log",
		            "Mode": "",
		            "RW": true,
		            "Propagation": "rprivate"
		        }
		    ],
		    "Config": {
		        "Hostname": "e23371d35b8e",
		        "Domainname": "",
		        "User": "",
		        "AttachStdin": false,
		        "AttachStdout": false,
		        "AttachStderr": false,
		        "ExposedPorts": {
		            "1414/tcp": {},
		            "9157/tcp": {},
		            "9443/tcp": {}
		        },
		        "Tty": false,
		        "OpenStdin": false,
		        "StdinOnce": false,
		        "Env": [
		            "COVERAGE_FILE=TestSecurityVulnerabilities.cov",
		            "EXIT_CODE_FILE=TestSecurityVulnerabilitiesExitCode",
		            "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
		            "LANG=en_US.UTF-8",
		            "AMQ_DIAGNOSTIC_MSG_SEVERITY=1",
		            "AMQ_ADDITIONAL_JSON_LOG=1",
		            "LOG_FORMAT=basic",
		            "MQ_DEV=true",
		            "MQ_ADMIN_PASSWORD=passw0rd"
		        ],
		        "Cmd": null,
		        "Image": "mqadvanced-server-dev:9.0.5.0-x86_64-ubuntu-16.04",
		        "Volumes": null,
		        "WorkingDir": "",
		        "Entrypoint": [
		            "bash",
		            "-c",
		            "source /etc/os-release \u0026\u0026 echo \"deb http://security.ubuntu.com/ubuntu/ ${VERSION_CODENAME}-security main restricted\" \u003e /etc/apt/sources.list \u0026\u0026 apt-get update 2\u003e\u00261 \u003e/dev/null \u0026\u0026 apt-get --simulate -qq upgrade"
		        ],
		        "OnBuild": null,
		        "Labels": {
		            "IBM_PRODUCT_ID": "98102d16795c4263ad9ca075190a2d4d",
		            "IBM_PRODUCT_NAME": "IBM MQ Advanced for Developers (Non-Warranted)",
		            "IBM_PRODUCT_VERSION": "9.0.5.0"
		        }
		    },
		    "NetworkSettings": {
		        "Bridge": "",
		        "SandboxID": "216d1f7d9530e7d67ce0cbe9e038c83b77ac964b09ccefc9b63a4eea601e121b",
		        "HairpinMode": false,
		        "LinkLocalIPv6Address": "",
		        "LinkLocalIPv6PrefixLen": 0,
		        "Ports": {},
		        "SandboxKey": "/var/run/docker/netns/216d1f7d9530",
		        "SecondaryIPAddresses": null,
		        "SecondaryIPv6Addresses": null,
		        "EndpointID": "",
		        "Gateway": "",
		        "GlobalIPv6Address": "",
		        "GlobalIPv6PrefixLen": 0,
		        "IPAddress": "",
		        "IPPrefixLen": 0,
		        "IPv6Gateway": "",
		        "MacAddress": "",
		        "Networks": {
		            "bridge": {
		                "IPAMConfig": null,
		                "Links": null,
		                "Aliases": null,
		                "NetworkID": "b0f7ebe61a150e0e336d8ed3c431e30eda620dc442413ea0ca78f41334878846",
		                "EndpointID": "",
		                "Gateway": "",
		                "IPAddress": "",
		                "IPPrefixLen": 0,
		                "IPv6Gateway": "",
		                "GlobalIPv6Address": "",
		                "GlobalIPv6PrefixLen": 0,
		                "MacAddress": ""
		            }
		        }
		    }
		}
	docker_api_test_util.go:183: Stopping container: e23371d35b8eb0315e9e1efde1121f124075c4b204f6ab4a76f9f2da01fd1732
	docker_api_test_util.go:191: Container stopped
	docker_api_test_util.go:196: Console log from container e23371d35b8eb0315e9e1efde1121f124075c4b204f6ab4a76f9f2da01fd1732:
		Inst gcc-5-base [5.4.0-6ubuntu1~16.04.9] (5.4.0-6ubuntu1~16.04.10 Ubuntu:16.04/xenial-security [amd64]) [libstdc++6:amd64 ]
		Conf gcc-5-base (5.4.0-6ubuntu1~16.04.10 Ubuntu:16.04/xenial-security [amd64]) [libstdc++6:amd64 ]
		Inst libstdc++6 [5.4.0-6ubuntu1~16.04.9] (5.4.0-6ubuntu1~16.04.10 Ubuntu:16.04/xenial-security [amd64])
		Conf libstdc++6 (5.4.0-6ubuntu1~16.04.10 Ubuntu:16.04/xenial-security [amd64])
	docker_api_test_util.go:204: Removing container: e23371d35b8eb0315e9e1efde1121f124075c4b204f6ab4a76f9f2da01fd1732
	docker_api_test.go:134: Expected no vulnerabilities, found the following:
		Inst gcc-5-base [5.4.0-6ubuntu1~16.04.9] (5.4.0-6ubuntu1~16.04.10 Ubuntu:16.04/xenial-security [amd64]) [libstdc++6:amd64 ]
		Conf gcc-5-base (5.4.0-6ubuntu1~16.04.10 Ubuntu:16.04/xenial-security [amd64]) [libstdc++6:amd64 ]
		Inst libstdc++6 [5.4.0-6ubuntu1~16.04.9] (5.4.0-6ubuntu1~16.04.10 Ubuntu:16.04/xenial-security [amd64])
		Conf libstdc++6 (5.4.0-6ubuntu1~16.04.10 Ubuntu:16.04/xenial-security [amd64])
```

--- a/install-mq.sh
+++ b/install-mq.sh
@@ -139,7 +139,7 @@ rm -rf ${DIR_EXTRACT}
 
 # Apply any bug fixes not included in base Ubuntu or MQ image.
 # Don't upgrade everything based on Docker best practices https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#run
-$UBUNTU && apt-get install -y gnupg gpgv libgcrypt20 perl-base --only-upgrade
+$UBUNTU && apt-get install -y gcc-5-base gnupg gpgv libgcrypt20 libstdc++6 perl-base --only-upgrade
 # End of bug fixes
 
 # Clean up cached files

